Gemfile.lock

```diff
@@ -48,32 +47,33 @@ GEM
     method_source (0.9.2)
     mini_portile2 (2.4.0)
     minitest (5.11.3)
-    money (6.13.2)
+    money (6.13.4)
```

Comment:
### [money](https://rubymoney.github.io/money)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 6.13.2 → 6.13.4

[_[change-log](https://github.com/RubyMoney/money/blob/master/CHANGELOG.md), [source-code](https://github.com/RubyMoney/money/)_]





<details>
<summary>Commits</summary>

A change of **12** commits. See the full changes on [the compare page](https://github.com/RubyMoney/money/compare/v6.13.2...v6.13.4).

These are the first 10 commits:
- (4340099) [Only include required files in the packaged gem (#844)](https://github.com/RubyMoney/money/commit/434009923d21e5a47e82a6f0da3d091b587e8e71)
- (422172a) [Use Currency::Loader directly without extending (#845)](https://github.com/RubyMoney/money/commit/422172a5c9bf76116d1ba55c7cd4ebaece3bbac8)
- (faa0323) [Add Money.with_rounding_mode as a replacement for Money.roud…](https://github.com/RubyMoney/money/commit/faa0323e45b608c3a0540be6d059b9fb0debde26)
- (1fc2c08) [Fix currency search for two digit currencies (#856)](https://github.com/RubyMoney/money/commit/1fc2c08216fb3d8d46d8733cc7db6d7f2f0b91d6)
- (c4b59a1) [Raise explicit errors for +/- operations (#852)](https://github.com/RubyMoney/money/commit/c4b59a10bfae6b449fb1745e44001e15d8622986)
- (f5d1f12) [Update version to 6.13.3](https://github.com/RubyMoney/money/commit/f5d1f12dbf74d7b859ec5859c033c64719c3ec1d)
- (421d0bc) [Typo in changelog (#861)](https://github.com/RubyMoney/money/commit/421d0bc0c60356182b56523ad5f9b549cb54aea6)
- (6cbddd4) [Fix typo in README (#865)](https://github.com/RubyMoney/money/commit/6cbddd48a608660c305d7f401ce6d9b327d6c7a1)
- (3bae6b0) [Fix typo in arithmetic spec's docstring (#864)](https://github.com/RubyMoney/money/commit/3bae6b073f38fc0bfb7d5bb9d19a090874606b9b)
- (e4d2a58) [Do not modify passed-in options in FormattingRules (#869)](https://github.com/RubyMoney/money/commit/e4d2a58bc36290ef9cc348f1cd76c7917a2c5783)


</details>